plugins/kafka-adapter/src/test/java/com.opendestro.kafkaAdapter/starter/ProducerTest.java

```diff
@@ -0,0 +1,40 @@
+package com.opendestro.kafkaAdapter.starter;
+
+import com.opendestro.kafkaAdapter.configuration.KafkaAdapterConf;
+import com.opendestro.kafkaAdapter.configuration.ProducerConfiguration;
+import com.opendestro.kafkaAdapter.util.Helper;
+import com.opendestro.kafkaAdapter.util.KafkaAdapterConsts;
+import com.opendestro.kafkaAdapter.util.Target;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.kafka.clients.producer.KafkaProducer;
+
+import org.junit.Test;
+
+import org.junit.Assert;
+
+import java.nio.file.Paths;
+
+public class ProducerTest {
```

Comment:
To properly test the logic you've written, you'll need to start a producer and a consumer, then make sure the consumer properly receives messages generated by the producer.